### ISSUE TITLE
Migration to MariaDB

### ISSUE THREAD

**Issue by Arrappathor** [circuscode/mathilda#1]
Hello! 
After migrating from mySQL to MariaDB newly imported tweets now appear at the bottom of the site. 
Resetting Mathilda arranges the tweets in correct order, but the new tweets again appear 'down under'...
Maybe a MariaDB-issue?
Anyway: fine tool, thanks a lot!

**Comment by circuscode** [comment]
Hey!
Thank you for reporting this issue.
Order of tweets will be determined by the MySQL database. The plugin requests the tweets from the database with a nested mysql select statement, which also includes descending order. It seems that this statement is not working completly with MariaDB.
Currently I do not have an environment with MariaDB in place for analyzing that. So I cannot provide an analysis on shortterm. But I will take this on my list for the next update. 
Will update here, if I have more news regarding this issue.
Best Regards

**Comment by Arrappathor** [comment]
OS was Raspbian 8 with MariaDB 10.0.30.
Thank you!

**Comment by circuscode** [comment]
Hey.
I have analyzed this issue. Sorry that I took so long.
My assumption above was right. The nested sql statement with the sorting is not supported by MariaDB. So I have changed the query that it is now working also with Maria.
You find the fix in the mathilda github repository. Just take the file mathilda_database.php and copy it into the plugin folder. That's all. I will prepare the update on wordpress.org in the meantime. I think, I can distribute it during next week.
Thank you very much for error reporting.
Best Regards
Marco
